mobify/astro-scaffold
# Appium

(brief description / PR title)

**Status**: _Ready for review_

**Reviewers**: @jansepar @jvaill @lizcross @jeremywiebe @mobify-derrick 
## Changes
- Sets up automated appium testing.
## How to test-drive this PR
- Checkout this repo
- Run `npm install -g appium`
- `cd app; npm install`
- From the root directory of the scaffold run `npm test`
- Xcode should build the app, and appium should run a basic test.
## TODO:
